src/backend/base/langflow/base/models/model.py

```diff
@@ -288,7 +288,7 @@ async def _handle_stream(self, runnable, inputs):
             else:
                 session_id = None
             model_message = Message(
-                text=runnable.stream(inputs),
+                text=await runnable.astream(inputs),
```

Comment:
`astream` returns a generator that gets consumed further down the process.
```suggestion
                text=runnable.astream(inputs),
```